app/Http/Requests/EntityRequest.php

```diff
@@ -32,6 +32,12 @@ public function entity()
                 $publicId = $this->$field;
             }
         }
+        if (! $publicId) {
```

Comment:
Can you explain why this change is needed?

Reply:
It's to handle the situation for custom modules where the entityType is different from the pluralized version.

Reply:
Thanks, understood